### ISSUE TITLE
Need stack? Why call onImageRendered?

### ISSUE THREAD

**Issue by jlopes90** [cornerstonejs/cornerstoneTools#1065]
Using cornerstoneTools 4.0

I just made a basic example of an image in https. There was an error that is missing "stack". Got to be the stack?

```
getLabelmaps3D.js:22 Uncaught TypeError: Cannot read property 'data' of undefined
    at Object.getLabelmaps3D [as labelmaps3D] (getLabelmaps3D.js:22)
    at onImageRenderedBrushEventHandler.js:21
    at HTMLDivElement.onImageRendered (imageRenderedEventDispatcher.js:25)
    at triggerEvent (triggerEvent.js:23)
    at drawImageSync.js:78
    at draw (enable.js:98)
```

### Codepen With Reproduction of Issue:
* [Basic Template](https://codepen.io/jlopes90/pen/jONxqXb)

**Comment by JamesAPetts** [comment]
Hmm, thanks for the Basic example, and also thanks for testing the waters with 4.0, your feedback is super helpful in this early stage!

The `segmentationModule` requires there be a stack for it to have some association between a 3D volume and the cornerstone images. But of course we shouldn't enforce this if segmentations aren't being used.

Here is a fix for now, by adding a stack. I'll add bail-outs for the renderer if stacks aren't being used.
https://codepen.io/JamesAPetts/pen/mdbLRWN

**Comment by JamesAPetts** [comment]
fixed in #1066. Feel free to re-open if you find other issues.
